### PR TITLE
Minor updates to the Developer Guide.

### PR DESCRIPTION
**Pull request by Sheikh-Umar**
I made minor updates to the Developer Guide.

### CODE REVIEW

File: docs/DeveloperGuide.adoc [hunk 1]
```diff
@@ -782,13 +782,14 @@ See this https://github.com/se-edu/addressbook-level4/pull/599[PR] for the step-
 
 *Target user profile*:
 
-* has a need to manage a significant number of contacts
+* Salespeople of companies
```
Comment (davidten) [inline docs/DeveloperGuide.adoc]: I think salespeople is enough

File: docs/DeveloperGuide.adoc [hunk 2]
```diff
@@ -782,13 +782,14 @@ See this https://github.com/se-edu/addressbook-level4/pull/599[PR] for the step-
...
+* has a need to manage a significant number of contacts into either Leads or Contacts
 * prefer desktop apps over other types
 * can type fast
 * prefers typing over mouse input
 * is reasonably comfortable using CLI apps
 
-*Value proposition*: manage contacts faster than a typical mouse/GUI driven app
+*Value proposition*: manage contacts faster and more efficiently than a typical mouse/GUI driven app
```
Comment (davidten) [inline docs/DeveloperGuide.adoc]: don't think this is needed

File: docs/DeveloperGuide.adoc [hunk 3]
```diff
@@ -800,7 +801,7 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 |Priority |As a ... |I want to ... |So that I can...
 |`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
 
-|`* * *` |user |add a new person |
+|`* * *` |user |add a new person to either Leads or Contacts|organise my sales targets better for future sales decisions
```
Comment (davidten) [inline docs/DeveloperGuide.adoc]: 'Add a new lead' don't add new person

File: docs/DeveloperGuide.adoc (hunk 2 above)
Comment (WoodyLau) [reply docs/DeveloperGuide.adoc]: Yeah, faster and more efficient are basically synonyms.

File: docs/DeveloperGuide.adoc (hunk 3 above)
Comment (WoodyLau) [reply docs/DeveloperGuide.adoc]: Splitting off into leads and contact sections probably works better. For formatting, there should be a space before each pipe.

File: docs/DeveloperGuide.adoc [hunk 4]
```diff
@@ -782,13 +782,14 @@ See this https://github.com/se-edu/addressbook-level4/pull/599[PR] for the step-
 
 *Target user profile*:
 
-* has a need to manage a significant number of contacts
+* Salespeople of companies
+* has a need to manage a significant number of contacts into either Leads or Contacts
```
Comment (WoodyLau) [inline docs/DeveloperGuide.adoc]: The double contacts seems a little odd. Maybe replace the first one with people, but it is less accurate.

File: docs/DeveloperGuide.adoc [hunk 5]
```diff
@@ -800,7 +801,7 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 |Priority |As a ... |I want to ... |So that I can...
 |`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
 
-|`* * *` |user |add a new person |
+|`* * *` |user |add a new lead to either Leads or Contacts|organise my sales targets better for future sales decisions
```
Comment (davidten) [inline docs/DeveloperGuide.adoc]: "add a new Lead or Contact"

**Pull request merged**